File: game_part/src/main/java/com/game/part/tmpl/IXlsxPacker.java
package com.game.part.tmpl;

import com.game.part.tmpl.type.AbstractXlsxTmpl;

/**
 * 打包接口
 * 
 * @author hjj2017
 * @since 2015/2/27
 * 
 */
public interface IXlsxPacker {
    /**
     * 打包模板对象
     *
     * @param tmplObj
     *
     */
    public void packUp(AbstractXlsxTmpl tmplObj);
}

File: client_wpf/Xgame.GamePart/Tmpl/IXlsxPacker.cs
using System;

using Xgame.GamePart.Tmpl.Type;

namespace Xgame.GamePart.Tmpl
{
    /// <summary>
    /// 打包接口
    /// </summary>
    public interface IXlsxPacker
    {
        /// <summary>
        /// 打包模板对象
        /// </summary>
        /// <param name="tmplObj"></param>
        void PackUp(BaseXlsxTmpl tmplObj);
    }
}
